docs/src/main/resources/ExtContents.adoc

```diff
@@ -2912,12 +2912,16 @@ _Similar to a Federated Source with the following exceptions:_
 ** _SiteService does not show this Source's information separate from DDF's._
 
 The procedure for implementing any of the source types follows a similar format:
-. Create a new class that implements the specified Source interface.
+. Create a new class that implements the specified Source interface and `ConfiguredService`.
 . Implement the required methods.
 . Create an OSGi descriptor file to communicate with the OSGi registry. (Refer to the OSGi Services  section.)
 .. Import DDF packages.
 .. Register source class as service to the OSGi registry.
 . Deploy to DDF. (Refer to the Working with OSGi - Bundles section.)
+[IMPORTANT]
+====
+The `factory-pid` property of the metatype must contain either service or source in the name
```

Comment:
The filter as written is a little more strict than that, isn't it? Unless the ldap filter is evaluated case-insensitively, the actual requirement is that the factory-pid must contain one of "Source|source|Service|service", literally.

If the ldap filter is actually case-insensitive, you could also simplify the filter query, naturally. ;)